### DISCUSSION TITLE
Send Emails with NestJS - https://notiz.dev/blog/send-emails-with-nestjs

### DISCUSSION THREAD

**Discussion by giscus** [notiz-dev/notiz#232] [General]
# Send Emails with NestJS - https://notiz.dev/blog/send-emails-with-nestjs

Create Email Templates and send them with nodemailer from your Nest application

https://notiz.dev/blog/send-emails-with-nestjs

**Comment by kellslte** [comment]
I love how straight forward and easy to follow your article is. Great job! 🙌

**Comment by GreenFlag31** [comment]
Don't use @nestjs-modules/mailer, it contains critical security vulnerability and should not be used in production !! Problem with a depedency vm2@3.9.19, which contains a critical security vulnerability and is not maintained anymore.

**Reply by crestoff** [reply]
you can override dependies and use it.

**Comment by ifernisimo** [comment]
Thank you , for this guide !

**Comment by benja971** [comment]
Love it !
